test/features/standup.feature

```diff
@@ -13,57 +8,58 @@ Feature: Standup-module
             | todayMessage     | "Today I'll do this"   |
         Then the response status code should be 201
         And the response should contain:
-            | id               | 123                        |
-            | discordUser      | "eddiehubber"              |
-            | yesterdayMessage | "Yesterday I did this"     |
-            | todayMessage     | "Today I'll do this"       |
-            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | documentId | "TYPE:ID" |
```

Comment:
Can we check for the `todayMessage` is `Today I'll do this` and that `yesterdayMessage` is empty or non existent?